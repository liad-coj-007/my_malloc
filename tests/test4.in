add 3 5
put 3 0 1
put 3 1 2
put 3 2 3
put 3 3 4
put 3 4 5
print_stats
realloc 3 10 4
put 4 5 0
put 4 6 0
put 4 7 0
put 4 8 0
put 4 9 0
print 4 0 9
print_stats
free 4
print_stats
