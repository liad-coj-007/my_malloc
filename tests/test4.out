Total free bytes: 0
Total used bytes: 20
Number of free blocks: 0
Number of allocated blocks: 1
[1, 2, 3, 4, 5, 0, 0, 0, 0, 0]
Total free bytes: 0
Total used bytes: 60
Number of free blocks: 0
Number of allocated blocks: 2
Total free bytes: 40
Total used bytes: 60
Number of free blocks: 1
Number of allocated blocks: 2